h/resources.py

```diff
@@ -169,6 +169,10 @@ def __init__(self, organization, request):
     def id(self):
         return self.organization.pubid  # Web-facing unique ID for this resource
 
+    @property
+    def default(self):
+        return self.organization == Organization.default(self.request.db)
```

Comment:
I'm concerned that this property lookup unnecessarily issues a db request when we could simply compare the pubid against __default__. Also what's the advantage of using a property here instead of an attribute? I believe this means each time .default is accessed it will run this comparison but this comparison would return the same value every time so why make it a property?

Reply:
D'oh! You're right, @hmstepanek and this was actually something I was thinking about when I was typing this code originally but forgot to mention it. I'll give this more thought.